pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "p5"
+version = "0.8.4"
+description = "p5 is a Python library that provides high level drawing functionality to help you quickly create simulations and interactive art using Python"
+authors = ["Abhik Pal <theabhikpal@gmail.com>"]
+readme = "README.rst"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+freetype-py = "^2.1.0"
+glfw = "^2.5.9"
+pillow = "^9.0.1"
+vispy = "^0.10.0"
+pyopengl-accelerate = "^3.1.7"
+pyopengl = "^3.1.6"
+requests = "^2.25.0"
+numpy = [    
```

Comment:
In the requirements.txt numpy is not tagged to a specific version (which is a bad practice already) 

I appreciate you pinning the version, but why different versions are pinned for different python version?

Reply:
This is needed because numpy version 1.21 is the last version that is python3.7 compatible.
Otherwise poetry can not figure out the dependency for python 3.7 if that makes sense ?

So by pinning it like this we can ensure compatibility on python3.7 and have a more up to date version for tthe python versiosn above.